buildozer/__init__.py

```diff
@@ -511,9 +511,9 @@ def check_application_requirements(self):
 
         # did we already installed the libs ?
         if exists(self.applibs_dir) and \
-           self.state.get('cache.applibs', '') == requirements:
-                self.debug('Application requirements already installed, pass')
-                return
+               self.state.get('cache.applibs', '') == requirements:
+            self.debug('Application requirements already installed, pass')
+            return
```

Comment:
I wish I didn't have to touch that, but for some reason (flake8 update?) the linter started complaining with:
```
buildozer/__init__.py:515:17: E117 over-indented
```
Also the diff is a bit messed up, but basically I'm simply changing from:
```python
        # did we already installed the libs ?
        if exists(self.applibs_dir) and \
           self.state.get('cache.applibs', '') == requirements:
                self.debug('Application requirements already installed, pass')
                return
```
to:
```python
        # did we already installed the libs ?
        if exists(self.applibs_dir) and \
               self.state.get('cache.applibs', '') == requirements:
            self.debug('Application requirements already installed, pass')
            return
```
So block stay the same, just fixing the over/under indentation

Reply:
i tend to prefer this form
```python
        if (
            exists(self.applibs_dir)
            and self.state.get('cache.applibs', '') == requirements
        ):
            self.debug('Application requirements already installed, pass')
            return
```
which i think is easier to read, but not a big deal.

Reply:
Thanks addressed!
Just the `and` part had to stay at the end of the line or `flake8` would complain

Reply:
ah, i guess either the version is not up to date with spec, or we have a config that goes against the current recommendations, but we can live with that :).

Reply:
I remember that rule changed recently indeed.
Once I'm doing providing a fully functional replacement for our VMs, I'll probably resume my work on fixing most of the linting